main-site/components/events-calendar-section/texts/DesktopTexts.tsx

```diff
@@ -40,15 +40,23 @@ const DesktopTexts: React.FC = ({isDay}) => {
               {event.description && <EventsP>{event.description}</EventsP>}
             </EventsLeftContainer>
             <EventsRightContainer>
+<<<<<<< HEAD
+              <EventsImage src={ event.image || MissingPhoto}></EventsImage>
+=======
               {event.image &&
               <EventsImage src={ event.image}></EventsImage>
               }
+>>>>>>> 68dd83ee2593b77f98d857147e44094f55659acc
```

Comment:
Whats this for?

Reply:
omg i thought all the merge conflicts were fixed, my bad. fixing now